docs-beta/src/css/custom.css

```diff
@@ -1,28 +1,36 @@
-/**
- * Any CSS included here will be global. The classic template
- * bundles Infima by default. Infima is a CSS framework designed to
- * work well for content-centric websites.
- */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
 
-/* You can override the default Infima variables here. */
-:root {
-  --ifm-color-primary: #a7cff7;
-  --ifm-color-primary-dark: #85c0f4;
-  --ifm-color-primary-darker: #5cb1f0;
-  --ifm-color-primary-darkest: #00a2ed;
-  --ifm-color-primary-light: #c6dffa;
-  --ifm-color-primary-lighter: #e3effc;
-  --ifm-color-primary-lightest: #ffffff;
-  --ifm-code-font-size: 95%;
-}
-
-.docusaurus-highlight-code-line {
-  background-color: rgba(0, 0, 0, 0.1);
-  display: block;
-  margin: 0 calc(-1 * var(--ifm-pre-padding));
-  padding: 0 var(--ifm-pre-padding);
-}
-
-html[data-theme='dark'] .docusaurus-highlight-code-line {
-  background-color: rgba(0, 0, 0, 0.3);
+@layer components {
+    .bodyBGColor {
+        @apply bg-primary;
+    }
+    .headerTitle {
+        @apply text-secondary text-6xl px-12 pb-4 pt-12;
+    }
+    .headerSubtitle {
+        @apply text-secondary text-3xl px-12 mb-4;
+    }
+    .button {
```

Comment:
You should aim to put these custom styles to the components themselves instead. React is already a component framework that can abstract these for you.

Reply:
Hmm, this one is a little controversial I think. :)

Looking at this from the design system perspective, there are some things we want to change once everywhere - for example, a body text title might be always primary color, bold and medium large.

But creating a 'title' component for a p tag with a string seems like overkill (and overhead at runtime), and manually maintaining the 3 or 4 body components styles is also not great...

I like this approach for standardizing the contracts for the 'common' items between components...